package.json

```diff
@@ -1,48 +1,54 @@
-    "start": "grunt",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+ssh://git@bitbucket.org/chill1/crypto-terminal.git"
-  },
-  "author": {
-    "name": "Charles Hill",
-    "email": "chill@degreesofzero.com"
-  },
-  "license": "AGPL-3.0",
-  "homepage": "https://bitbucket.org/chill1/crypto-terminal#readme",
-  "dependencies": {
-    "async": "2.3.0",
-    "backbone": "1.3.3",
-    "backbone.localstorage": "2.0.0",
-    "bignumber.js": "4.0.0",
-    "bitcoinjs-lib": "3.0.2",
-    "browserify": "14.3.0",
-    "grunt": "1.0.1",
-    "grunt-contrib-concat": "1.0.1",
-    "grunt-contrib-connect": "1.0.2",
-    "grunt-contrib-copy": "1.0.0",
-    "grunt-contrib-cssmin": "2.0.0",
-    "grunt-contrib-uglify": "2.2.0",
-    "grunt-mocha-test": "0.13.2",
-    "grunt-newer": "1.2.0",
-    "handlebars": "4.0.6",
-    "jquery": "3.2.1",
-    "load-grunt-config": "0.19.2",
-    "mocha": "3.4.2",
-    "moment": "2.18.1",
-    "open-sans-fontface": "https://github.com/chill117/open-sans/archive/1.4.2.tar.gz",
-    "qrcode-generator": "1.2.0",
-    "selenium-standalone": "6.5.0",
-    "sjcl": "1.0.6",
-    "time-grunt": "1.4.0",
-    "underscore": "1.8.3",
-    "webdriverio": "4.8.0"
-  }
-}
+    "name": "crypto-terminal",
+    "version": "1.0.0",
+    "description": "Accept payments with Bitcoin, Litecoin, Monero, and more!",
+    "main": "index.js",
+    "scripts": {
+        "start": "grunt",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+ssh://git@bitbucket.org/chill1/crypto-terminal.git"
+    },
+    "author": {
+        "name": "Charles Hill",
+        "email": "chill@degreesofzero.com"
+    },
+    "license": "AGPL-3.0",
+    "homepage": "https://bitbucket.org/chill1/crypto-terminal#readme",
+    "dependencies": {
+        "async": "2.3.0",
+        "backbone": "1.3.3",
+        "backbone.localstorage": "2.0.0",
+        "bignumber.js": "4.0.0",
+        "bitcoinjs-lib": "3.0.2",
+        "browserify": "14.3.0",
+        "cordova-browser": "^4.1.0",
+        "grunt": "1.0.1",
+        "grunt-contrib-concat": "1.0.1",
+        "grunt-contrib-connect": "1.0.2",
+        "grunt-contrib-copy": "1.0.0",
+        "grunt-contrib-cssmin": "2.0.0",
+        "grunt-contrib-uglify": "2.2.0",
+        "grunt-mocha-test": "0.13.2",
+        "grunt-newer": "1.2.0",
+        "handlebars": "4.0.6",
+        "jquery": "3.2.1",
+        "load-grunt-config": "0.19.2",
+        "mocha": "3.4.2",
+        "moment": "^2.19.1",
+        "open-sans-fontface": "https://github.com/chill117/open-sans/archive/1.4.2.tar.gz",
+        "qrcode-generator": "1.2.0",
+        "selenium-standalone": "6.5.0",
+        "sjcl": "1.0.6",
+        "time-grunt": "1.4.0",
+        "underscore": "1.8.3",
+        "webdriverio": "4.8.0"
+    },
+    "cordova": {
+        "platforms": [
+            "browser"
+        ]
+    }
+}
```

Comment:
Can you revert the original indentation in this file so that we can see which lines you added more clearly? Now it looks like the whole file was changed. Can create a separate PR to change the indentation if we want.